handler.go

```diff
@@ -74,6 +74,30 @@ func (m *multiHandler) Flush() {
 	}
 }
 
+// FilteredHandler constructs a Handler that processes Measures with `filter` before forwarding to `h`
+func FilteredHandler(h Handler, filter func([]Measure) []Measure) Handler {
+	return &filteredHandler{handler: h, filter: filter}
+}
+
+type filteredHandler struct {
+	handler Handler
+	filter  func([]Measure) []Measure
+}
+
+func (h *filteredHandler) HandleMeasures(time time.Time, measures ...Measure) {
+	filteredMeasures := h.filter(measures)
+
+	if len(filteredMeasures) == 0 {
```

Comment:
This doesn't seem obviously correct... it's possible there are downstream handlers that might want to get called even if all measurements have been filtered out.

Reply:
Yeah, I agree. It's simpler and probably less prone to future problems to remove that short-circuit. Good suggestion!